storyblok/components.json

```diff
@@ -2192,6 +2192,14 @@
               "value": "SE_APARTMENT_STUDENT_RENT",
               "name": "SE_APARTMENT_STUDENT_RENT"
             },
+            {
+              "value": "SE_ACCIDENT",
+              "name": "SE_ACCIDENT"
+            },
+            {
+              "value": "SE_ACCIDENT_STUDENT",
+              "name": "SE_ACCIDENT_STUDENT"
+            },
```

Comment:
Do we need two different options here? I was under the impression that the student accident insurance is exactly the same as the regular one? But I might have misunderstood of course 😅